File: betweenTwoSets/betweenTwoSets.js
/*
 * Complete the 'getTotalX' function below.
 *
 * The function is expected to return an INTEGER.
 * The function accepts following parameters:
 *  1. INTEGER_ARRAY a
 *  2. INTEGER_ARRAY b
 */

function getTotalX(a, b) {
	var num = 0, count = 0, check = 0, err = 0;
	
	if(a.length >= 1 && a.length <= 10 &&
	   b.length >= 1 && b.length <= 10) {

		do {

			num++;

			for(var i = 0; i < a.length; i++) {
				if(a[i] >= 1 && a[i] <= 100) {
					if(num >= a[i] && num%a[i] == 0)
						check++;
				}
				else
					err++;
			}

			for(var j = 0; j < b.length; j++) {
				if(b[j] >= 1 && b[j] <= 100) {
					if(num <= b[j] && b[j]%num == 0)
						check++;
				}
				else
					err++;
			}

			if(check == (a.length + b.length) - err)
				count++;

			check = 0;
			err = 0;

		}while(num <= 100);

		return count;

	}

}

function main() {
    
	const arr = [2, 6];
	const brr = [24, 36];
    const total = getTotalX(arr, brr);

    console.log(total + '\n');

}

main();

File: betweenTwoSets/betweenTwoSets.py
def getTotalX(a, b) :
	
    check = 0
    num = 0
    err = 0
    count = 0

    if len(a) >= 1 and len(a) <= 10 and len(b) >= 1 and len(b) <= 10 :
        while num <= 100 :

            num += 1

            for x in a :
                if x >= 1 and x <= 100 :
                    if num >= x and num%x == 0 :
                        check += 1
                else :
                    err += 1

            for y in b :
                if y >= 1 and y <= 100 :
                    if num <= y and y%num == 0 :
                        check += 1
                else :
                    err += 1

            if check == (len(a) + len(b)) - err :
                count += 1

            check = 0
            err = 0

        return count



def main() :
    
    arr = [2, 4]
    brr = [16, 32, 96]
    total = getTotalX(arr, brr)

    txt = "{}\n"
    print(txt.format(total))

main()
    